build.sbt

```diff
@@ -550,7 +542,8 @@ lazy val sharing = (project in file("sharing"))
     name := "delta-sharing-spark",
     commonSettings,
     scalaStyleSettings,
-    releaseSettings,
+    // TODO re-enable release once 1.3.2 is available
```

Comment:
We will reenable this + the tests once there is a 1.3.2 delta sharing client release (was blocked on Spark 4.0 being available). We have tested this using a staged release of 1.3.2 and tests were passing